apps/api/src/app/inbox/utils/notification-mapper.ts

```diff
@@ -52,6 +51,11 @@ const mapSingleItem = ({
     },
     channelType: channel,
     tags,
+    redirect: cta.data.url
+      ? {
+          url: cta.data.url,
+        }
+      : undefined,
```

Comment:
we can extend the functionality later when this field is an object, for example, add `target: '_blank'` etc